tests/hikari/impl/test_interaction_server.py

```diff
@@ -21,21 +21,61 @@
 # SOFTWARE.
 import asyncio
 import contextlib
+import typing
```

Comment:
Redundant import?

Reply:
typing isn't being imported anywhere else there, any new tests I'm writing will be typed btw